content/developer/_index.md

```diff
@@ -4,3 +4,29 @@ weight: 2
 layout: playbook
 ---
 
+The OMSF Developer Playbook is a comprehensive guide designed to help scientists and developers create high-quality, maintainable software for the scientific community. This playbook provides practical guidance on best practices that can be adopted at various stages of your project's lifecycle, from inception through long-term maintenance.
+
+## Purpose and Audience
+
+Our primary goal is to increase developer literacy within the scientific community. By sharing established best practices, we aim to raise the overall quality of software in the space, leading to more reliable, reproducible, and usable scientific tools.
+
+This playbook is intended for:
+- Scientists developing software for their research
+- Developers working on scientific applications
```

Comment:
Interesting how much emphasis you put on science here. TBH, I think this has a much broader audience, although we're framing it a little more for scientific software developers. But part of the reason we really need little detailed text, and can get away with quick summaries, is that the ideas have been covered so many time before.